packages/web-app-files/src/components/AppBar/SelectedResources/BatchActions.vue

```diff
@@ -24,6 +24,17 @@
       </oc-button>
     </template>
     <oc-grid v-if="displayBulkActions" gutter="small">
+      <div v-if="canDownloadAsArchive">
```

Comment:
Currently, if you only have one file selected (e.g. an image), it downloads this image inside a `.tar` file. Can we add a distinction here on whether a single file or a folder / multiple files are selected and handle the download interaction accordingly? :)

Reply:
yes, good idea 👍

Reply:
Please check if the implementation reflects your idea properly.